src/interpret.jl

```diff
@@ -173,12 +173,7 @@ function bypass_builtins(frame, call_expr, pc)
         if isa(tme, Compiled)
             fargs = collect_args(frame, call_expr)
             f = to_function(fargs[1])
-            fmod = parentmodule(f)::Module
-            if fmod === JuliaInterpreter.CompiledCalls || fmod === Core.Compiler
-                return Some{Any}(Base.invokelatest(f, fargs[2:end]...))
-            else
-                return Some{Any}(f(fargs[2:end]...))
-            end
```

Comment:
Maybe we want to keep this code around (like with commented out), and leave some comment that says ideally we want to figure out the root problem here and remove the added `invokelatest`.

Reply:
Done.